tests/test_models/phelps_deaton.py

```diff
@@ -1,65 +1,71 @@
-"""Define example model specifications."""
+"""Example specifications of the Phelps-Deaton model."""
 
 import jax.numpy as jnp
 
-RETIREMENT_AGE = 65
-N_CHOICE_GRID_POINTS = 500
-N_STATE_GRID_POINTS = 100
+# ======================================================================================
+# Numerical parameters and constants
+# ======================================================================================
 
+N_GRID_POINTS = {
+    "states": 100,
```

Comment:
```suggestion
    "wealth": 100,
```
I think it should be more useful to be explicit rather than general here.